reana_workflow_controller/rest/workflows.py

```diff
@@ -237,7 +247,7 @@ def get_workflows(paginate=None):  # noqa
                 "status": workflow.status.name,
                 "user": user_uuid,
                 "created": workflow.created.strftime(WORKFLOW_TIME_FORMAT),
-                "progress": get_workflow_progress(workflow),
+                "progress": get_workflow_progress(workflow, include_progress),
```

Comment:
```suggestion
                "progress": get_workflow_progress(workflow, include_progress=include_progress),
```

For the sake of readability.